apps/dynatrace/dynatrace-operator2.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: dynatrace
+spec:
+  releaseName: dynatrace-operator
+  interval: 5m
+  chart:
+    spec:
+      chart: dynatrace-operator
+      version: 0.4.2
+      sourceRef:
+        name: dynatrace-operator2
```

Comment:
Do we need the 2 reference(s)?

Reply:
Using the 2nd reference and the helm chart I'm pulling down this name see helm repo file.

Reply:
I would prefer to keep all without a 2 incoming helm repo etc